src/regression_tests/regression_test.test.ts

```diff
@@ -0,0 +1,138 @@
+import fetchMock from 'jest-fetch-mock';
+import {
+  createUser,
+  getBalance,
+  getEnvVariables,
+  getPaymailDomain,
+  getTransactions,
+  RegressionTestConfig,
+  RegressionTestUser,
+  removeRegisteredPaymail,
+  sendFunds,
+} from './utils';
+
+const MINIMAL_FUNDS_PER_TRANSACTION = 2;
+// Two minute timeout for transaction tests
+const TEST_TIMEOUT_MS = 120000;
```

Comment:
I'm not a fun of the approach when the value is tightly coupled with the comment.
This can lead to situations where the timeout value is changed, but the comment is not updated accordingly.

```suggestion
const TEST_TIMEOUT_MS = 2 * 60 * 1000;
```

Reply:
Fair point, changed